raiden/raiden_service.py

```diff
@@ -998,7 +999,7 @@ def _initialize_channel_fees(self) -> None:
                 token_address=token_address,
             )
 
-            for channel in channels:
+            for channel in [c for c in channels if c.fee_schedule is None]:
```

Comment:
Simplify here. Maybe add the comprehension to a variable before this line.

Reply:
Also, document why we do this.

Reply:
All the function does is `_initialize_channel_fees`, but ok.